huaweicloud/data_source_huaweicloud_obs_bucket_object_test.go

```diff
@@ -0,0 +1,224 @@
+package huaweicloud
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/terraform"
+	"github.com/huaweicloud/golangsdk/openstack/obs"
+)
+
+func TestAccHuaweiCloudObsBucketObjectDataSource_content(t *testing.T) {
```

Comment:
seems the `TestAccHuaweiCloudObsBucketObjectDataSource_content` and `TestAccHuaweiCloudObsBucketObjectDataSource_source` are the same testing case for data source.